test/unit/lodash.spec.js

```diff
@@ -0,0 +1,275 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var chance = require('chance');
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash Collection training', function () {
+    describe('countBy', function () {
```

Comment:
countBy: we don't have any tests for iterator function, I can just hardcore the answer without even providing iterator. Types of collections are not tested, too.